### ISSUE TITLE
Cannot start SCH AF3 quest Seeing Blood-red

### ISSUE THREAD

**Issue by helixhamin** [DerpyProjectGroup/topaz#650]
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened
- [x] I have checked the commit log to see if the issue has been resolved since my server was last updated
- [x] I have read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I have specified what branch this happens on branch: EDITME

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 
Requirements met (level 56 SCH).
Waited one game day after completing Downward Helix.
Searched the script, and do not see this in Erlene's code.

**Comment by RAIST5150** [comment]
Yeah... according to the globals file, that quest has not been scripted/deemed playable yet:

https://github.com/LandSandBoat/server/blob/base/scripts/globals/quests.lua

There has been a lot of focus on the TAU jobs recently (PUP just got a big update, think some AF fixes for BLU?)... not sure if anyone was actively working on WotG/SOA jobs atm though.  They were all flagged as a questionable state last I recall.
